.travis.yml

```diff
@@ -42,21 +42,33 @@ branches:
     - master
 
 matrix:
+  include:
+    - rvm: jruby-9.1.17.0
+      env: DRIVER=ruby REDIS_BRANCH=3.0 JRUBY_OPENSSL=0.9.21
+    - rvm: jruby-9.1.17.0
+      env: DRIVER=ruby REDIS_BRANCH=3.2 JRUBY_OPENSSL=0.9.21
+    - rvm: jruby-9.1.17.0
+      env: DRIVER=ruby REDIS_BRANCH=stable JRUBY_OPENSSL=0.9.21
+  allow_failures:
+      # Using jruby-openssl 0.10.0, we get NPEs in jruby tests: https://github.com/redis/redis-rb/issues/756
+      # jruby-openssl 0.9.21 required tests have been added above in the include section.
+    - rvm: jruby-9.1.17.0
+      env: DRIVER=ruby      REDIS_BRANCH=3.0
+    - rvm: jruby-9.1.17.0
```

Comment:
I don't think it's worth running test we know will fail. If the fix for the jruby test suite is to use `jruby-openssl < 0.10` then let's add this to the Gemfile / gemspec and that's it.